test/e2e-multicluster-test/multicluster_test.go

```diff
@@ -396,5 +398,33 @@ var _ = Describe("Test multicluster scenario", func() {
 			err = k8sClient.Delete(hubCtx, &checkApp)
 			Expect(err).Should(BeNil())
 		})
+
+		It("Test deploy multi-cluster application with target", func() {
+			By("apply application")
+			app := &v1beta1.Application{
+				ObjectMeta: v12.ObjectMeta{Namespace: namespace, Name: "test-app-target"},
```

Comment:
nit: replace the generated import name to a elaborative named import such as "metav1", "clusterv1", etc